lib/knapsack_pro/config/ci/heroku.rb

```diff
@@ -0,0 +1,31 @@
+module KnapsackPro
+  module Config
+    module CI
+      class Heroku < Base
+        def node_total
+          # not provided
```

Comment:
It would be great to be able to read from Heroku ENV info how many parallel CI nodes we have. For instance we have 5 parallel jobs on Heroku then the value for `node_total` should be `5`.